install/minikube-installation-guide.md

```diff
@@ -32,6 +32,10 @@ This installation guide has only been verified under a Minikube Linux installati
 > - This installation guide may not work for macOS installations of Minikube, due to the lack of
 nested virtualization support on that platform.
 > - This installation guide has not been tested on a Windows installation.
+> - Kata under Minikube does not currently support Kata Firecracker (`kata-fc`), due to
+>   lack of a block based storage driver availability. Although the `kata-fc` binary will be
```

Comment:
Nit: I think the word "availability" is redundant?

> due to the lack of a block based storage driver.

Reply:
hmm, given the thread that spawned this update, I quite like the way it confirms why there is a lack of driver... implies that it is not something one can just go update/fix trivially. I'll change it if you really want though...?

Reply:
It just doesn't read well to me, but I'm struggling to think how to improve the wording. I'll ack and see if @klynnrif has any thoughts...

Reply:
np. I simplified and dropped the whole section about why it will not work, and now just note it won't work... (and, the reasons why may change or be incomplete, so just saying 'does not work' should have a longer shelf life, until the day it does work...)